docs/resources/user.md

```diff
@@ -37,12 +35,11 @@ In addition to all arguments above, the following attributes are exported:
 * `user_id` - The ID of the user
 * `status` - Status of the user.  Either `invited` or `subscribed`
 
-
 Import
 ------
 
-Users can be imported using the user ID, e.g.
+Users can be imported using their user email address, e.g.
 
 ```
-$ terraform import rollbar_user.foo 238101
+$ terraform import rollbar_user.some_dev some_dev@company.com
```

Comment:
Lined up the example resource name and email address in the example above - clearer to the reader.